### PR TITLE
Update release version 0.5

### PR DESCRIPTION
**Pull request by Amulyam24**
**What this PR does / why we need it**:
Update release version 0.5

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

/area provider/ibmcloud

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note
Update release version 0.5
```

### CODE REVIEW

**Review by Prajyot-Parab — APPROVED** [review]
/lgtm

File: metadata.yaml [hunk 1]
```diff
@@ -14,3 +14,6 @@ releaseSeries:
   - major: 0
     minor: 4
     contract: v1beta1
+  - major: 0
+    minor: 5
+    contract: v1beta2
```
Comment (mkumatag) [inline metadata.yaml:19]: this should be `v1beta1` the version of a capi api we have contract with

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: metadata.yaml (hunk 1 above)
Comment (Amulyam24) [reply metadata.yaml]: Got it, Addressed 👍🏼

**Review by mkumatag — APPROVED** [review]
/lgtm

**Pull request merged**